sufia-models/app/models/concerns/sufia/generic_file/csv.rb

```diff
@@ -0,0 +1,23 @@
+module Sufia
+  module GenericFile
+    module CSV
+      def to_csv
+        ::CSV.generate do |csv|
```

Comment:
Should this method also spit out a header row so that folks know what the various values are for?